### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by pratap-panabaka**
Implemented views, Add posts and comments to the database

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    </div>
+    <% end %>
+
+    <% if post.comments.length != 0 %>
+    <div class=rectangle>
+        <% post.most_recent_comments.each do |comment| %>
+        <%= comment.author.name %>: <%= comment.text %><br><br>
+        <% end %>
+    </div>
+    <% end %>
```
Comment (Munsa1) [inline app/views/posts/index.html.erb]: - [x] Kindly follow the given design as this is a core requirement

Your design
image

Given design
image

- Kindly move the number of posts to the left of the design as it is in the given design
- Also move the comments and likes counters to the left as it is in the given design

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (pratap-panabaka) [reply app/views/posts/index.html.erb]: @Munsa1 Thank you for your review of this PR.
I have implemented the changes required.

image

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @PRATAP-KUMAR , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**